src/DurableTask.SqlServer/SqlConnectionFactory.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Threading.Tasks;
+
+using Azure.Core;
+using Azure.Identity;
+
+using Microsoft.Data.SqlClient;
```

Comment:
Just for stylistic consistency, it would be great if you could move these `using` statements inside the `namespace` block.